ephios/plugins/simpleresource/forms.py

```diff
@@ -17,9 +16,8 @@ class Meta:
         }
 
     def __init__(self, *args, shift, **kwargs):
+        kwargs.setdefault("prefix", "simple_resource")
         self.shift = shift
-        self.helper = FormHelper()
-        self.helper.include_media = False
```

Comment:
@jeriox why was this here? Didn't find another use of it...

Reply:
was used to fix https://github.com/ephios-dev/ephios/issues/853 but I don't understand how

Reply:
jsi18n offers to handle the loading of the translation catalogue for you by using the django forms media interface. This includes the JS file, but from a path that doesn't match to what we are doing. In the base, we are importing the catalogue ourselves, so it works everywhere else. Using the form helper and the template tag here meant that crispy also injected the form media, which meant the erroneous JS path. Setting this to false prevents crispy from doing that and everything is fine. So as long as you don't use the crispy tag all should be fine

Reply:
Alright, so as we are not using the tag here, this is fine. Sounds like another reason to rethink our form rendering though. 🙈